Resources/Prototypes/Body/Prototypes/arachnid.yml

```diff
@@ -18,6 +18,7 @@
         stomach: OrganArachnidStomach
         liver: OrganArachnidLiver
         kidneys: OrganArachnidKidneys
+      organSlots: [ powerCell, charger ] # DeltaV augments
```

Comment:
do not do this
instead add surgery code to create an organ slot then add them for torso and arms (they have to displace something after all)
then undo all changes to prototype reader and prototypes

Reply:
issue: IPCs already have power cells and the augments should work with that and it would be weird to let you implant separate power cells/chargers in them

Reply:
blacklist ipcs for the power surgery (and ideally the arm one would be less surgical and more mechanical)